packages/dashboard/src/pages/Wallets/components/WalletStateBar/index.tsx

```diff
@@ -64,7 +64,9 @@ export const WalletStateBar = memo(() => {
     const wallet = useWallet()
     const networkDescriptor = useNetworkDescriptor()
     const providerDescriptor = useProviderDescriptor()
-    const { value: pendingTransactions = [] } = useRecentTransactions(TransactionStatusType.NOT_DEPEND)
+    const { value: pendingTransactions = [] } = useRecentTransactions({
```

Comment:
Replace empty arrays that might cause re-render in hooks or components with constant `EMPTY_LIST`.